compiler/one-cmds/onelib/utils.py

```diff
@@ -134,15 +134,16 @@ def safemain(main, mainpath):
         sys.exit(255)
 
 
-def run(cmd, err_prefix=None, logfile=None):
+def run(cmd, err_prefix=None, logfile=None, env=None):
```

Comment:
This change is to turn on logs for circle_mpqsolver.

Other tools control logs using `--verbose` option. Do you have any reason for directly passing env variables?

Reply:
> This change is to turn on logs for circle_mpqsolver.
> 
> Other tools control logs using `--verbose` option. Do you have any reason for directly passing env variables?

@jinevening 
Yes. It's just for logging. Would it be better to implement logging by `verbose` option?

Reply:
Yes. It would be better to keep the interface consistent.

Reply:
> Yes. It would be better to keep the interface consistent.

Ok. Thank you.